src/mame/ibm/thinkpad8xx.cpp

```diff
@@ -0,0 +1,98 @@
+// license:BSD-3-Clause
+// copyright-holders:
+/***********************************************************************************************************
+Skeleton driver for IBM Thinkpad Power Series.
+The IBM ThinkPad Power Series (800/820/821/822/823/850/851/860) is a laptop series from the ThinkPad line
+that was manufactured by IBM. It is based on the PowerPC architecture.
+All of the PowerPC ThinkPads could run Windows NT 3.51 and 4.0, AIX 4.1.x, and Solaris Desktop 2.5.1 PowerPC
+Edition. It is also possible to run certain PowerPC versions of Linux on the 800 Series.
+
+Hardware for the 850 model:
+-SCSI hard disk and SCSI CD-ROM drive (with a NCR 53C810 PCI-SCSI I/O Processor).
+-Two PC Card expansión slots (with a Ricoh RF53C366L PC Card interface controller and a MAX780 Dual-Slot PCMCIA Analog Power Controller).
+-Video: 
+   -IBM 85G7815 (by Seiko/Epson).
+   -Western Digital WD90C24A SVGA LCD controller.
+   -Two Hitachi HM51S4260 262144 x 16bit DRAM (1MB of video display memory).
+   -10.4" 640×480 or 800×600 screen.
+-Video capture:
+   -Brooktree Bt812 NTSC/PAL to RGB/YCrCb Decoder.
+   -Two Hitachi HM530281 high speed 331776 x 8bit Frame buffer DRAM.
+   -ASCII V7310AS (アスキー, Asukii) Video Capture Device.
+-Crystal CS4231 16bit stereo codec for audio.
+-Two DRAM DIMMs slots.
+-Hitachi H8/338 (HD6473388) for main board supervisión.
+-CPU:
+    -IBM PowerPC 603e @ 100MHz (PPCI603eFC100BPQ).
+    -Two 32k x 36bits IBM043614 burst SRAM (256k L2 cache total).
+    -IDT71216 240K (16K x 15bit) cache-tag RAM.
+    -33.333 MHz xtal (tripled for 100MHz system clock).
+-Dallas DS1585S RTC.
+-Intel S82378ZB PCIset.
+-National Semiconductor DP87322VF (SuperI/O III, Floppy Disk Controller with Dual UARTs, Enhanced Parallel Port, and IDE Interface).
+-Motorola XPC105ARX66CD (PowerPC PCI Bridge/Memory Controller).
+-Other ICs: S-MOS 85G7814, S-MOS 85G2680
+
+More info: http://oldcomputer.info/portables/tp850/ibm_ppc_thinkpad_redbook.pdf
+
+***********************************************************************************************************/
+
+#include "emu.h"
+#include "cpu/h8/h8325.h"
+#include "cpu/powerpc/ppc.h"
+#include "emupal.h"
+#include "screen.h"
+#include "softlist_dev.h"
+#include "speaker.h"
+
+
+namespace {
+
+class thinkpad8xx_state : public driver_device
+{
+public:
+	thinkpad8xx_state(const machine_config &mconfig, device_type type, const char *tag)
+		: driver_device(mconfig, type, tag)
+		, m_maincpu(*this, "maincpu")
+	{ }
+
+
+	void thinkpad8xx(machine_config &config);
```

Comment:
This should be `thinkpad850` – the other models have different hardware.

Reply:
Fixed!

Reply:
Why clang on Linux fails with 'Driver "thinkpad850" found for source file "ibm/thinkpad850.cpp" but defined in source file "ibm/thinkpad8xx.cpp"' Can you please help?

Reply:
Because your change to mame.lst is wrong:
```
@source:ibm/thinkpad850.cpp
thinkpad850                     // IBM Thinkpad 850
```

`@source` directive doesn’t match actual source file name (see @angelosa – that validation task really does work).

Reply:
Argggg... 

Fixed, Thanks!!